evidence/clangenb/0003-retention-rank-1-202501.md

```diff
@@ -0,0 +1,55 @@
+# Evidence-0003: Retention at Rank 1
+
+|                 |                                 |
+| --------------- |---------------------------------|
+| **Report Date** | Date of submission (2025/01/08) |
+| **Submitted by**| Christian Langenbacher          |
+
+
+## Member details
+
+- Matrix username: `@clang:matrix.org`
+- Polkadot address: `16YCL3UVpVWQLGW3p3Zx4k5WAEp9W1DwdDnxAbyAaPxVxnp3`
+- Current rank: `1`
+- Date of initial induction: `2024/03/27`
+- Date of last report: `2024/10/15`
+- Area(s) of Expertise/Interest: `Parachain Builder, Runtime, Clients`
+
+
+## Reporting period
+
+- Start date: 2024/10/15
+- End date: 2025/01/08
+
+
+## Evidence
+My activity mostly consisted of smaller PRs in the polkadot-sdk, and commenting on some issues in the polkadot-sdk and fellowship runtime repositories. A full list can be found at the end of the document, but I will list a few things here:
+* Substantial contributions to the migration of the benchmarking v1 syntax to the benchmarking v2 syntax, see the [PRs](https://github.com/paritytech/polkadot-sdk/pulls?q=is%3Apr+reviewed-by%3Aclangenb+created%3A2024-10-15..2025-01-08+). Note: Some of the contributions belong to the last reporting [period](https://collectives.subsquare.io/fellowship/referenda/231#4), but the majority of work has been done in this period.
+* Investigated how to enable admin transfers in the pallet-assets for frozen assets, see [comment](https://github.com/paritytech/polkadot-sdk/issues/3992#issuecomment-2553395720).
+* I made my first appearance on the fellowship OpenDev call, and I aspire to be a regular participant from now on.
+
+## Voting record
+*Provide your voting record in relation to required thresholds for your rank.* 
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
+|---|---|----------------------|----------------------------|---|
+|I  |90%   | N/A                  | No elligible referenda     |  |
+|II |80%   | N/A                  |                            |  |
+|III|70%   | 100%                 |                            |  |
+|IV |60%   | 90%                  |                            |  |
+|V  |50%   | 80%                  |                            |  |
+|VI |40%   | 70%                  |                            |  |
+
+
+## Misc
+
+- [ ] Question(s): 
+
+- [ ] Concern(s): 
+
+- [ ] Comment(s):
+
+### General activity in reporting period
```

Comment:
This section seems duplicate. Why not move this stuff into `Evidence` section ?

Reply:
I have seen this pattern in another Evidence PR that was deemed to be idiomatic. The Evidence section was the actual important work, and in this section is just general activity in this period to high-light ecosystem activity, which might individually be too insignificant to mention above.

But maybe in my particular case, it doesn't make a big difference, and I should have just merged the two.

This was the PR I took for reference: https://github.com/polkadot-fellows/Evaluations/pull/81/files